compiler/ballerina-lang/src/main/java/io/ballerina/projects/JBallerinaBackend.java

```diff
@@ -453,7 +454,10 @@ private Manifest createManifest() {
     private void copyJar(ZipArchiveOutputStream outStream, JarLibrary jarLibrary,
             HashMap<String, JarLibrary> copiedEntries, HashMap<String, StringBuilder> services) throws IOException {
 
-        ZipFile zipFile = new ZipFile(jarLibrary.path().toFile());
+        String pkgName = jarLibrary.packageName().get();
```

Comment:
Shall we get the organization name along with the package name.
Package name can be same with different organizations
```
ballerina/io;
abc/io;
```

Reply:
Checked it and it is working for other organization names as well. It seems like even though it was named "packageName", it contains the org name as well.

![image](https://github.com/ballerina-platform/ballerina-lang/assets/105668612/88d02cc9-b4af-434d-a1b8-565cdf55ac09)